content/yaml-publishing/google-cloud-storage.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Google Cloud Storage
+description: How to publish build artifacts to Google Cloud Storage using codemagic.yaml
+weight: 14
+---
+
+In order to publish your generated artifacts to Google Cloud Storage:
+
+1. Create a new Service Account and generate a JSON file with **Storage Object Creator** role. Save your JSON file to [environment variable](https://docs.codemagic.io/variables/environment-variable-groups/). 
```

Comment:
Can we give a bit more detail around this?

Maybe in the first step explain how to create the required service account as follows:

Log in to your Google Cloud platform [console](https://console.cloud.google.com/).

Create a new service account under IAM & Admin > Service Accounts
Give Storage Object Admin access to the newly created service account.
Create a key for the service account. Choose key type as JSON. This will cause a JSON key file to be downloaded.

Then in another step, say that they should create an environment variable called GCLOUD_STORAGE_KEY and save the contents of the Service Account JSON to the the variable. Then link to the docs as you have done already.